setup.py

```diff
@@ -13,13 +13,14 @@
 
 install_requires = [
     'packaging>=20,<22',
-    "numpy>=1.18.0,<1.20.0;python_version<'3.7'",
-    "numpy>=1.20.0,<2;python_version>='3.7'",
-    'pandas>=1.1.3,<2',
-    'scikit-learn>=0.24,<2',
-    'torch>=1.8.0,<2',
-    'torchvision>=0.9.0,<1',
```

Comment:
I think that this can be removed, right ?

Reply:
what was it here for before? So long as `SDV` can still run fine I think it's alright

Reply:
Yes, all the tests seem to run smoothly.